Makefile

```
@@ -66,6 +66,9 @@ PROMETHEUS_NAMESPACE=gotenberg
 PROMETHEUS_COLLECT_INTERVAL=1s
 PROMETHEUS_DISABLE_ROUTE_LOGGING=false
 PROMETHEUS_DISABLE_COLLECT=false
+SENTRY_DSN=
+SENTRY_SEND_DEFAULT_PII=true
+SENTRY_ENVIRONMENT=production
```

```suggestion
SENTRY_ENVIRONMENT=
```

Should be the default values here 😄 

Also it misses `sentry-flush-timeout`